sql/V007__messages_create.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS messages(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    sendtime TEXT NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    content TEXT NOT NULL,
+    user TEXT NOT NULL,
+    discriminator INTEGER NOT NULL,
+    channel INTEGER NOT NULL,
+    server TEXT NOT NULL
+);
```

Comment:
I like what you've done with the `messages` table, but these changes should really live in their own PR. This PR was only intended to address the bug in the `!weather` command.